webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/AnsibleVariableOverridesTable.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  TableComposable,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from '@patternfly/react-table';
+
+import {
+  formatValue,
+  formatSourceAttr,
+} from './AnsibleVariableOverridesTableHelper';
+
+const AnsibleVariableOverridesTable = ({ variables, hostAttrs }) => {
+  const columns = [
+    __('Name'),
+    __('Ansible Role'),
+    __('Type'),
+    __('Value'),
+    __('Source attribute'),
+  ];
+
+  return (
+    <TableComposable variant="compact">
+      <Thead>
+        <Tr>
+          {columns.map((col, idx) => (
+            <Th key={idx}>{col}</Th>
+          ))}
+        </Tr>
+      </Thead>
+      <Tbody>
+        {variables.map((variable, idx) => (
```

Comment:
There are two components at least with the same key due to using `index` as a key.
[react docs](https://reactjs.org/docs/lists-and-keys.html) doesn't recommend using `index` as a key because it does not represent the data correctly and therefore can break dynamic data.